test/unit-tests.c

```diff
@@ -101,6 +101,21 @@ test_dma_map_without_fd(void **state __attribute__((unused)))
     assert_int_equal(0, handle_dma_map_or_unmap(&vfu_ctx, size, true, &fd, 0, &r));
 }
 
+static void dma_map_cb(UNUSED vfu_ctx_t *vfu_ctx, uint64_t iova, uint64_t len,
+                uint32_t prot)
+{
+    if (iova == 0xcafebabe) {
+        assert(len == 0x1000);
+        assert(prot == (PROT_READ|PROT_WRITE));
```

Comment:
nit: not need for parenthesis plus space between `|` operator

Reply:
w/o parenthesis compiler warnings (-Werror=parentheses). But anyways now with assert_int_equal() it doesn't matter.